### ISSUE TITLE
Support for Polygons with holes

### ISSUE THREAD

**Issue by redfrexx** [geoscripting/03_scientific_programming#2]
Adapt the `MyPolygon` class so that it supports polygons with holes

**Comment by AnnaBooki** [comment]
Hi :)
hast du zufällig auch bei def __init__ etwas verändert?
Mein Code sieht aktuell so aus
'''
class MyPolygon:
    def __init__(self, coordinates):
        self.coordinates = coordinates

    def polyhole(self):
        import shapely
        ext_coords, int_coords = self.coordinates.split("],")[0][1]
        pgn = shapely.Polygon(ext_coords, holes=[int_coords])
        return pgn

'''

**Comment by lgroe** [comment]
Ja, ich hab es bisher so, dass die inneren und äußern Koordinaten getrennt als Parameter eingegeben werden, mit None als default value für die inneren Koordinaten, falls das Polygon kein Loch hat. 

bisher sieht es so aus: 

```
class MyPolygon:

    def __init__(self, outer_coordinates, inner_coordinates = None):
        if inner_coordinates == None:
            self.outer_coordinates = outer_coordinates
        else:
            self.outer_coordinates = outer_coordinates
            self.inner_coordinates = inner_coordinates


```

**Comment by AnnaBooki** [comment]
Cool habe es eben ähnlich verändert mit exterior und interior, das mit None hatte ich noch nicht
